platform/ios/PolyPodApp/PolyPod/Authentication.swift

```diff
@@ -35,21 +35,48 @@ class Authentication {
         UserDefaults.standard.set(true, forKey: Authentication.disableCheckKey)
     }
     
-    func setUp(_ completeAction: @escaping (Bool) -> Void) {
-        authenticateLocally(reason: "auth_prompt_set_up") { success in
+    func setUp(_ reAuth: Bool, _ completeAction: @escaping (Bool) -> Void) {
```

Comment:
Since it does not seem like an obvious thing that setup is called with a Boolean that indicates whether it should use re-authentication, wouldn't this be a bit clearer if you used an external label for the argument? As a matter of fact, probably both should use an external label.